test/python/transpiler/test_passmanager_run.py

```diff
@@ -0,0 +1,118 @@
+        qr1:-----(+)--.--------  -> 2
+                      |
+        qr2:---------(+)--.----  -> 3
+                          |
+        qr3:-------------(+)---  -> 5
+
+        device:
+        0  -  1  -  2  -  3  -  4  -  5  -  6
+
+              |     |     |     |     |     |
+
+              13 -  12  - 11 -  10 -  9  -  8  -   7
+        """
+        qr = QuantumRegister(4, 'qr')
+        circuit = QuantumCircuit(qr)
+        circuit.h(qr[0])
+        circuit.cx(qr[0], qr[1])
+        circuit.cx(qr[1], qr[2])
+        circuit.cx(qr[2], qr[3])
+
+        coupling_map = FakeMelbourne().configuration().coupling_map
+        basis_gates = FakeMelbourne().configuration().basis_gates
+        initial_layout = [None, qr[0], qr[1], qr[2], None, qr[3]]
+
+        pass_manager = level_1_pass_manager(TranspileConfig(
+            basis_gates=basis_gates,
+            coupling_map=CouplingMap(coupling_map),
+            initial_layout=Layout.from_qubit_list(initial_layout),
+            seed_transpiler=42,
+            optimization_level=1))
+        new_circuit = pass_manager.run(circuit)
+
+        for gate, qargs, _ in new_circuit.data:
+            if isinstance(gate, CnotGate):
+                self.assertIn([x.index for x in qargs], coupling_map)
+
+    def test_default_pass_manager_two(self):
+        """Test default_pass_manager.run(circuitS).
+
+        circuit1 and circuit2:
+        qr0:-[H]--.------------  -> 1
+                  |
+        qr1:-----(+)--.--------  -> 2
+                      |
+        qr2:---------(+)--.----  -> 3
+                          |
+        qr3:-------------(+)---  -> 5
+
+        device:
+        0  -  1  -  2  -  3  -  4  -  5  -  6
+
+              |     |     |     |     |     |
+
+              13 -  12  - 11 -  10 -  9  -  8  -   7
+        """
+        qr = QuantumRegister(4, 'qr')
+        circuit1 = QuantumCircuit(qr)
+        circuit1.h(qr[0])
+        circuit1.cx(qr[0], qr[1])
+        circuit1.cx(qr[1], qr[2])
+        circuit1.cx(qr[2], qr[3])
+
+        circuit2 = QuantumCircuit(qr)
+        circuit2.h(qr[0])
+        circuit2.cx(qr[0], qr[1])
+        circuit2.cx(qr[1], qr[2])
+        circuit2.cx(qr[2], qr[3])
+
+        coupling_map = FakeMelbourne().configuration().coupling_map
+        basis_gates = FakeMelbourne().configuration().basis_gates
+        initial_layout = [None, qr[0], qr[1], qr[2], None, qr[3]]
+
+        pass_manager = level_1_pass_manager(TranspileConfig(
+            basis_gates=basis_gates,
+            coupling_map=CouplingMap(coupling_map),
+            initial_layout=Layout.from_qubit_list(initial_layout),
+            seed_transpiler=42,
+            optimization_level=1))
+        new_circuits = pass_manager.run([circuit1, circuit2])
+
+        for new_circuit in new_circuits:
+            for gate, qargs, _ in new_circuit.data:
+                if isinstance(gate, CnotGate):
+                    self.assertIn([x.index for x in qargs], coupling_map)
```

Comment:
```
self.assertEqual(new_circuits[0], new_circuits[1])
```?

since the actual mapping is tested in the other test and this one is just checking that the two produce the same result.

as an aside is it possible to test that this was indeed done in parallel?

Reply:
I put different circuits in 132e94c7